backend/package.json

```diff
@@ -16,6 +16,7 @@
     "cors": "^2.8.5",
     "dotenv": "^16.5.0",
     "express": "^5.1.0",
+    "lru-cache": "^11.1.0",
```

Comment:
This could be a good thing to add to the readme for tech stack

Reply:
Added!